src/model/scripts/gptj_thai_tokenizer/merge_tokenizers.py

```diff
@@ -0,0 +1,7 @@
+from openthaigpt_pretraining_model.GPTJ_TH_tokenizer.merge import merge
+from openthaigpt_pretraining_model.GPTJ_TH_tokenizer.constants import (
+    OUTPUT_HF_DIR,
+)
+
+tokenizer = merge()
```

Comment:
Can you make thai_tokenizer available as an argument when running this file? If the user did not selected thai_tokenzier the default should be `flax-community/gpt2-base-thai`

Example:
```python
import argparse

parser = argparse.ArgumentParser(description='Process some integers.')

parser.add_argument('integers', metavar='N', type=int, nargs='+',
                    help='an integer for the accumulator')

parser.add_argument('--sum', dest='accumulate', action='store_const',
                    const=sum, default=max,
                    help='sum the integers (default: find the max)')

args = parser.parse_args()
print(args.accumulate(args.integers))
```

Reply:
Done